blueprints/hr_assistant/webex_bot_server.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+import os
```

Comment:
Should we refactor this as a component in MindMeld @vijay120 ?

Reply:
Debatable, since its not core to MindMeld but is a nice to have integration.